Microsoft.Toolkit.Uwp.UI.Controls/Graph/SharePointFiles/FileSelectedEventArgs.cs

```diff
@@ -0,0 +1,28 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using Microsoft.Graph;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.Graph
+{
+    /// <summary>
+    /// Arguments relating to a file selected event of SharePointFiles control
+    /// </summary>
+    public class FileSelectedEventArgs : EventArgs
+    {
```

Comment:
Should this be publicly creatable? Generally eventargs aren't, and there's an implicit public constructor here

Reply:
In .NET before UWP most event args were publicly constructable, it makes it easier to mock and test events.  I'm not sure why they changed this with UWP.

Reply:
done